Release/2.0

FNQuispe opened the pull request:
⚠ BREAKING CHANGES:

Montos: Ahora todos los montos se manejan sin decimales, se mandan como numero entero teniendo en cuenta que los últimos dos son los decimales que se tomaran en el sistema.
✨ Features

Operaciones pago PCI: Se agregó la opción de realizar pagos PCI.
Operaciones con tokenización externa con pago PCI: Se agregó la opción de tokenización externa con pago PCI.
Operaciones con 3DS: Se agrego la opción de pagar con autenticación 3DS.

sbuonincontro-rb commented:
estas mergeando develop a master, incluyendo los cambios que mencionas. Eso es correcto?

juan1756 commented:
### **Todos los cambios que se incluyen son los siguientes:**
Anulacion de devolucion mpos
Sub agrupador
Tipos de error CardErrorCode
Campo Channel de String a Enum
Refactor Exceptions Java6
Manejo de error Validation para Refunds(PYA)

lschinini committed:
add classes auth3dsData y auth3dsResponse. Add auth3dsResponse into PaymentResponse.

lschinini committed:
add classes auth3dsData y auth3dsResponse. Add auth3dsResponse into PaymentResponse. (#11)

FNQuispe committed:
feat(payments): [FPP11-3352] add 3ds to payments

FNQuispe committed:
Merge branch 'develop' into feature/FPP11-3352-sdk-java-pago-con-3ds

# Conflicts:
#	src/main/java/com/decidir/sdk/dto/auth3ds/Auth3dsResponse.java
#	src/main/java/com/decidir/sdk/payments/Payment.java

FNQuispe committed:
Merge pull request #12 from payway-ar/feature/FPP11-3352-sdk-java-pago-con-3ds

Feature/fpp11 3352 sdk java pago con 3ds

lschinini committed:
bugfix(TOKEN):[FPP11-3703]: delete method tokenCs. Unification of Token with cs (#13)

lschinini committed:
bugfix(TOKEN):[FPP11-3703]: delete classes tokenCs and tokenRequest

lschinini committed:
fix(PAYMENT):[FPP11-4086] add PaymentAuth3dsResponseException. fix catch exception. (#14)

lschinini committed:
bugfix(Payment travel):[FPP11-3703]: update class DepartureDate (#15)

BartoDev committed:
feat(CardTokenData): [FPP11-5652] more fields are added. (#17)

feat(FraudDetectionPciModel): [FPP11-5652] class added.
feat(installmentSpv): [FPP11-5652] class added.
feat(Payment): [FPP11-5652]  Description field are added.
feat(PaymentPciRequest): [FPP11-5652] more fields are added.
feat(PaymentRequest): [FP11-5352] Spv object are added.
feat(PaymentResponse): [FPP11-5652] more fields are added.
feat(PCICardData): [FPP11-5652] more fields are added.
feat(Spv): [FPP11-5652] Class added.

LuisAlejandroBernalRomero committed:
Se implementó tokenización interna

LuisAlejandroBernalRomero committed:
Merge branch 'master' into feature/FPP11-5899-tokenizacion-interna-en-sdk-java

LuisAlejandroBernalRomero committed:
Se borran archivos innecesarios

LuisAlejandroBernalRomero committed:
Merge branch 'feature/FPP11-5899-tokenizacion-interna-en-sdk-java' of github.com:payway-ar/sdk-java-ventaonline into feature/FPP11-5899-tokenizacion-interna-en-sdk-java

LuisAlejandroBernalRomero committed:
Merge pull request #18 from payway-ar/feature/FPP11-5899-tokenizacion-interna-en-sdk-java

Feature/fpp11 5899 tokenizacion interna en sdk java

LuisAlejandroBernalRomero committed:
fix(FPP11-6290): Se implementó que retornara los mensajes de error de Decidir

LuisAlejandroBernalRomero committed:
Merge pull request #19 from payway-ar/bugfix/FPP11-6290-manejo-de-errores-en-el-mensaje-y-el-codigo-de-la-respuesta

Bugfix/fpp11 6290 manejo de errores en el mensaje y el codigo de la respuesta

EzequielAlg committed:
added checkout hash to java sdk

EzequielAlg committed:
added dist folder

EzequielAlg committed:
merge conflicts

EzequielAlg committed:
added checkout hass functionality

EzequielAlg committed:
added documentation

EzequielAlg committed:
build.gradle modificado

EzequielAlg committed:
modified dist

EzequielAlg committed:
Merge pull request #20 from payway-ar/feature/FPP11-5374-checkout

Feature/fpp11 5374 checkout

EzequielAlg committed:
added payment description to CheckoutRequest

EzequielAlg committed:
Merge branch 'develop' of https://github.com/payway-ar/sdk-java-ventaonline into feature/FPP11-5374-checkout

EzequielAlg committed:
Merge pull request #21 from payway-ar/feature/FPP11-5374-checkout

Feature/fpp11 5374 checkout

EzequielAlg committed:
added new exception

EzequielAlg committed:
Merge branch 'develop' of https://github.com/payway-ar/sdk-java-ventaonline into feature/FPP11-5374-checkout

EzequielAlg committed:
Merge pull request #22 from payway-ar/feature/FPP11-5374-checkout

Feature/fpp11 5374 checkout

FNQuispe committed:
Update README.md

FNQuispe committed:
Update README.md

FNQuispe committed:
Merge pull request #23 from payway-ar/update-readme

Update README.md

lschinini committed:
feat(CONFIRMACION PAYMENT): update type of card data. String to Object (#24)
